milestones/3_human_brain_mapping_journal_submission/experiments/ex_2_pli_generation_optimization/run_pli_generation.bash

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# This script work by iteratively giving as parameter the epoch and the p_id to sbatch (40 combination of them in total)
+# This params are given as a parameter to the job.sl that we've defined as the first input parameters
+# Meaning that we should call this bash script like this:
+# ./run_pli_generation.bash [job_name.sl]
+# Where job_name.sl is the name of the slurm file that we want to iteratively batch for scheduling.
+
+# Variable we will iterate around
+EPOCHS=("eyesclosed_1" "induction" "emergence_first" "emergence_last" "eyesclosed_8")
+P_IDS=("MDFA03" "MDFA05" "MDFA06" "MDFA07" "MDFA10" "MDFA11" "MDFA12" "MDFA15" "MDFA17")
```

Comment:
I think I was looking for this in one of my earlier comments on **ex_2_pli_generation_optimization/README.md** 
Here, it is really clear